src/components/Chat/ChatPage.js

```diff
@@ -1,17 +1,21 @@
 import React, { Component } from 'react';
 import {
+  AppState,
   AsyncStorage,
```

Comment:
'AsyncStorage' is defined but never used
'ListView' is defined but never used
'Platform' is defined but never used
'RefreshControl' is defined but never used
'ScrollView' is defined but never used
'TextInput' is defined but never used
'TouchableHighlight' is defined but never used